### PR TITLE
Sign up UI & functionality

### PR DESCRIPTION
**Pull request by sharathnair9999**
Completed Sign in  and Sign up UI with Functionality (Redux Store yet to set up)
Fixed few styling issues and cleaned the code by taking out the functions into outer helper files

### CODE REVIEW

**Review by shivanigangadharan — APPROVED** [review]
Changes look good to go 👍🏼

**Review by prathmeshjagtap — APPROVED** [review]
The code looks clean Liked that you have kept SVG in constant folder

File: src/components/TopNav/TopNav.jsx [hunk 1]
```diff
@@ -1,7 +1,50 @@
-import React from "react";
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom";
+import { logout } from "../../pages/Auth/helpers";
```
Comment (bajajcodes) [inline src/components/TopNav/TopNav.jsx:4]: Use absolute import.

File: src/components/TopNav/components/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const Button = ({ onClick, className, children }) => {
+  return (
+    <div
```
Comment (bajajcodes) [inline src/components/TopNav/components/Button.jsx:5]: Rule: Semantic HTML over anything.

File: src/components/Brand.js [hunk 3]
```diff
@@ -1,22 +1,25 @@
...
-      />
-      <span
-        className={` ${(name || full) && "block"} ${
-          logo && "hidden"
-        } font-satisfy font-bold  text-transparent text-4xl p-1 bg-clip-text bg-gradient-to-r from-cta-dark to-accent-1`}
-      >
-        SocialEase
-      </span>
-    </header>
+    <Link to={"/feed"}>
```
Comment (bajajcodes) [inline src/components/Brand.js:7]: Suggestion: link can go inside header tag.

File: src/pages/Auth/Login.jsx [hunk 4]
```diff
@@ -20,15 +21,9 @@ const Login = () => {
   const loginUser = async (e, enteredEmail, password) => {
```
Comment (bajajcodes) [inline src/pages/Auth/Login.jsx:21]: I guess error handling is weak here.

File: src/components/TopNav/components/Button.jsx (hunk 2 above)
Comment (sharathnair9999) [reply src/components/TopNav/components/Button.jsx:5]: thanks for pointing it out. `rafce` spoiled me 😂

File: src/components/Brand.js (hunk 3 above)
Comment (sharathnair9999) [reply src/components/Brand.js:7]: done

**Review by KanekarSnehal — APPROVED** [review]
No issues found, you can merge it

File: src/pages/Auth/Login.jsx (hunk 4 above)
Comment (sharathnair9999) [reply src/pages/Auth/Login.jsx:21]: Form is taking care of it. i will add formik in the coming days after working on post functionality

File: src/components/TopNav/TopNav.jsx (hunk 1 above)
Comment (sharathnair9999) [reply src/components/TopNav/TopNav.jsx:4]: yeah did it

**Comment by bajajcodes** [comment]
Left few comments, code is good to merge post consideration.

**Pull request merged**